install_config/install/deploy_router.adoc

```diff
@@ -609,6 +609,58 @@ $ echo "    router metrics at: http://<ip>:9090/consoles/haproxy.html "
 ====
 
 
+=== Preventing Connection Failures During Restarts ===
+
+If you connect to the router while the proxy is reloading, there is a
+small chance that your connection will end up in the wrong network
+queue and be dropped.  There is kernel work going on to remove that
+problem, but in the meantime it is possible to work around the problem
+by installing *_iptables_* rules to prevent connections during the
+reload window.  However, doing so means that the router needs to run
+with elevated privilege so that it can manipulate *_iptables_* on the
+host.  It also means that connections that happen during the reload
+are temporarily ignored and will have to retransmit their connection
+start, lengthening the time it takes to connect, but preventing
+connection failure.
+
+Because of these reasons, and because the risk is small, we do not
+recommend people enable this option most of the time.  However, if you
+decide you must try to prevent this case from happening, you can make
+the router use *_iptables_* by changing the service account, and setting
+an environment variable on the router.
+
+*Use a Privileged SCC*
+
+When creating the router, allow it to use the privileged SCC.  Note
+that this gives the router user the ability to create containers with
+root privileges on the nodes.
+----
+$ oadm policy add-scc-to-user privileged -z router
+----
+
+*Patch the Router Deployment Config to Create a Privileged Container*
+
+Now that the router use can create privileged containers, we need to
+make the router deployment config use the power so that the router can
+set the iptables rules it needs.  This patch will change the router
+deployment config so that the containter that is created runs as root.
+----
+$ oc patch dc router -p '{"spec":{"template":{"spec":{"containers":[{"name":"router","securityContext":{"privileged":true}}]}}}}'
+----
+
+*Tell the Router to use iptables*
+
+Set the option on the router deployment controller (if you used a
```

Comment:
Consider reworking:
    Set the option on the router deployment controller. If you used a non-default name for the router, you will need to change *_dc/router_* accordingly: